app/stac_api/models.py

```diff
@@ -6,13 +6,18 @@
 
 # import botocore.exceptions # Un-comment with BGDIINF_SB-1625
 import multihash
+from multihash import to_hex_string
```

Comment:
any particular reason for importing multihash twice in different ways?

Reply:
Historical, but I'll change it as follow:
```python
from multihash import encode as multihash_encode
from multihash import to_hex_string
```
Which makes more sense